pkg/argparser/args.go

```diff
@@ -86,23 +81,10 @@ func analyzeArgs(am *argMap, args string) {
 	}
 }
 
-func GetArgs(args string, state *state.HelmState, opts *GetArgsOptions) []string {
+func CollectArgs(args string) []string {
 	argsMap := newArgMap()
-
-	if len(args) > 0 {
-		analyzeArgs(argsMap, args)
-	}
-
-	if len(state.HelmDefaults.Args) > 0 {
-		analyzeArgs(argsMap, strings.Join(state.HelmDefaults.Args, " "))
-	}
-
-	if len(state.HelmDefaults.DiffArgs) > 0 && opts != nil && opts.WithDiffArgs {
-		analyzeArgs(argsMap, strings.Join(state.HelmDefaults.DiffArgs, " "))
-	}
-
+	analyzeArgs(argsMap, args)
```

Comment:
Where did the analyzeArgs call for `state.HelmDefaults.Args` go now?

Reply:
Ah okay so you moved the entire GetArgs to pkg/app/app.go now? (But why?!

Reply:
for solving import cycle issues. Make the `argparser` package more clean.

Reply:
@mumoshu

Reply:
I think this is a good direction, argparser decouples the business logic.